### DISCUSSION TITLE
Including periodic translations invariance in the SchNet implementation.

### DISCUSSION THREAD

**Discussion by fedeotto** [pyg-team/pytorch_geometric#4942] [Q&A]
I'm wondering if it is straightforward to modify the `SchNet` torch geometric [implementation](https://pytorch-geometric.readthedocs.io/en/latest/_modules/torch_geometric/nn/models/schnet.html) by including periodic conditions, as described in the [paper](https://aip.scitation.org/doi/pdf/10.1063/1.5019779)? I guess the actual implementation relies on an earlier version of the paper..?

**Comment by rusty1s** [comment]
I haven't looked into that for a while, but I don't think it should be complicated to add. The current implementation replicates the trained benchmark-models of `schnetpack` from [here](https://schnetpack.readthedocs.io/en/stable/getstarted/benchmarks.html).

**Reply by fedeotto** [reply]
Hi Matthias and thanks for the kind answer.

Looking at the implementation of this [paper](https://arxiv.org/abs/2110.06197), I've found that these people have obtained that by modifying `radius_graph` function in the case of `DimeNet++`. I still have to figure out exactly what they do though.
